src/controllers/wizardfinishpage.js

```diff
@@ -6,17 +6,11 @@ define(["loading"], function(loading) {
             url: ApiClient.getUrl("Startup/Complete"),
             type: "POST"
         }).then(function() {
-            Dashboard.navigate("dashboard.html");
             loading.hide();
+            window.location.href = "index.html";
```

Comment:
Did you test this redirect? @thornbill mentioned it might need `web` at the front, but if this works fine I'd rather leave it this way.

Reply:
i tested but @thornbill why `web` needed?

Reply:
I haven’t had a chance to test this pr, but I know going directly to `/index.html` on my server returns an error. It must either be `/` or `/web/index.html` for the page to load correctly.

Reply:
@thornbill wouldn't it be fine since the location property can be a relative link?

Reply:
Tested and it works as intended!